### PR TITLE
use /container/<container_id>/ for container metrics

### PR DESCRIPTION
**Pull request by malnick**

### CODE REVIEW

File: plugins/plugin.go [hunk 1]
```diff
@@ -156,13 +157,51 @@ func (p *Plugin) setEndpoints() error {
...
+		if err != nil {
+			logrus.Errorf("Encountered error reading response body, %s", err.Error())
+			return err
+		}
+
+		if err := json.Unmarshal(body, &containers); err != nil {
+			return err
+		}
+
+		for _, c := range containers {
```
Comment (nickbp) [inline plugins/plugin.go]: Looks like this is only fetched once at startup? What if containers are added/removed?

Comment (malnick) [inline plugins/plugin.go]: If you call `.Metrics()` in your plugin this will call `getEndpoints()` again.

Comment (malnick) [inline plugins/plugin.go]: As long as you call `.Metrics()` over and over this should refresh.

Comment (malnick) [inline plugins/plugin.go]: https://github.com/dcos/dcos-metrics/blob/a3ebfb6ecca8f71e93625bc5cd7838b8e3ee5eb0/plugins/plugin.go#L111

Comment (nickbp) [inline plugins/plugin.go]: oic makes sense

**Pull request merged**